.github/workflows/build.yaml

```diff
@@ -1,13 +1,18 @@
 name: Build binaries
 
 on:
+  pull_request_target:
   pull_request:
   push:
     branches:
       - master
     tags:
       - "v*"
 
+concurrency:
```

Comment:
Is it related to `The dependabot generated PRs don't have access to secrets`? I can't connect the dots how it's related.

Reply:
concurrency avoids running the same pipeline multiple times; and with dependabot generating/rebasing the PRs, it is just a good practice to use concurrency.

Reply:
👍